partiql-cli/src/main/kotlin/org/partiql/cli/Main.kt

```diff
@@ -237,7 +243,42 @@ internal class MainCommand : Runnable {
             // val connector = LocalPlugin.create(root.toPath())
             // return mapOf("default" to connector)
         }
+
+        if (env != null) {
+            return listOf(loadPartiQL(env!!))
+        }
+
         // Derive a `default catalog from stdin (or file streams)
+        return listOf(loadIon())
+    }
+
+    private fun loadPartiQL(env: File): Catalog {
+        if (!env.exists()) {
+            error("File not exists! Please check the env file path.")
```

Comment:
nit: grammar and let's also print out the env file

```suggestion
            error("File '$env' does not exist! Please check the env file path.")
```